DEVELOPER.md

```diff
@@ -1,106 +1,159 @@
 # Developers Guide
 
-<!-- TOC -->
-* [Developers Guide](#developers-guide)
-  * [Prerequisite Software](#prerequisite-software)
-  * [Prerequisite Services](#prerequisite-services)
-  * [Environment Setup](#environment-setup)
-    * [React Native & Android Emulator](#react-native--android-emulator)
-    * [Create Android emulator](#create-android-emulator)
-  * [Development](#development)
-    * [Workspace Setup](#workspace-setup)
-    * [Running in an Android Emulator](#running-in-an-android-emulator)
-    * [Debugging in Intellij and Android Emulator](#debugging-in-intellij-and-android-emulator)
-    * [Debugging in VSCode and Android Emulator](#debugging-in-vscode-and-android-emulator)
-  * [Source Code Information](#source-code-information)
-<!-- TOC -->
-
-## Prerequisite Software
-
-Before you can build and test, you must install and configure the
+* [Prerequisite software](#prerequisite-software)
+* [Prerequisite services](#prerequisite-services)
+* [Cloning and initializing submodule](#cloning-and-initializing-submodule)
+* [React Native setup](#react-native-setup)
+* [Installing npm modules](#installing-npm-modules)
+* [Configuration](#configuration)
+* [Running in an Android emulator](#running-in-an-android-emulator)
+* [Troubleshooting and debugging](#troubleshooting-and-debugging)
+
+## Prerequisite software
+
+Before you can proceed with building and testing the BC Wallet app, you must install and configure the
 following products on your development machine:
 
 * [Git](https://git-scm.com/)
 * [Node.js](https://nodejs.org) & [npm](https://docs.npmjs.com/cli/) - (version specified in the `engines` field
   of [./app/package.json](./app/package.json))
   > **Tip**: use [nvm](https://github.com/nvm-sh/nvm) to install node & npm. It helps to easily switch node & npm
   version for each project.
-* Android SDK and Android Emulator
-  > **Tip**: It is recommended to install the [Android Studio](https://developer.android.com/studio), it comes with
-  android sdk,
-  > device manager to install emulators.
-* JDK 8 or 11 (Preferably OpenJDK or ZuluJDK)
 
-## Prerequisite Services
+## Prerequisite services
 Requires the following services to be up and running.
 
 - [Von Network](https://github.com/bcgov/von-network)
```

Comment:
Might want to remove the line about von network if we're recommending that users use BCovrin instead